### ISSUE TITLE
Incorrect fc weights in [Resnet34, Arcface, MS1MV3] pretrained models

### ISSUE THREAD

**Issue by zzzzz314314** [deepinsight/insightface#2056]
Hello,
for the [Resnet34, Arcface, MS1MV3], the fc weights might not be correct. The number of classes is inconsistent (91180!=93431) in both baidu and one drive.

https://pan.baidu.com/s/1CL-l4zWqsI1oDuEEYVhj-g#list/path=%2Fsharelink3494273403-838107067808247%2Farcface_torch%2Fms1mv3_arcface_r34_fp16&parentPath=%2Fsharelink3494273403-838107067808247

https://onedrive.live.com/?authkey=%21AFZjr283nwZHqbA&id=4A83B6B633B029CC%215579&cid=4A83B6B633B029CC

Would it be possible that you re-upload them?
Many thanks

**Comment by nttstar** [comment]
the softmax fc weights should not be saved in our pretrained models.

**Comment by thinh-re** [comment]
Hi, thank you for your amazing work.
I would like to have the Softmax FC weights for incremental learning experiment.

To be more clearer, @zzzzz314314 don't mean the softmax FC weights in pretrained model `backbone.pt`. He means that softmax FC weights in all the files `rank_0_softmax_weight.pt`, `rank_1_softmax_weight.pt`, ..., `rank_7_softmax_weight.pt` combine together, which is 91180 classes. But MS1MV3 dataset has totally 93431 classes.

**Comment by ElisonSherton** [comment]
Hi @zzzzz314314 

Did you get any update about this? I also looked at the resnet50 versions and over there too, the softmax fc weights combined together leads to only 91180 classes instead of 93431 classes.

Alternatively, @nttstar , is there a dataset of 91180 classes for ms1mv3 which was cleaned further by `insightface` which we are not aware of; or which hasn't been publicly provided?

Thanks & Regards,
Vinayak Nayak.

**Comment by supertx** [comment]
any upload message ? asking from 2024/6/3
